src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -826,6 +826,39 @@ In case of PAM users, the login file should not be modified as password authenti
 NOTE: Similarly to the LDAP authentication, there is a fallback mechanism for login authentication, with the difference that it is always activated.
 A user defined in the ProActive login file always has precedence over the same user defined on the Linux system.
 
+== Task Termination Behavior
+
+=== ProActive Node Graceful Task Termination (SIGTERM) at Killing
+
+The task termination timeout is a cleanup timeout for each task (executed on this Node)
+after it was killed (through the Scheduler REST API or the Scheduler web portal).
+The task termination timeout can be setup on each ProActive Node individually.
+It can be set with the "proactive.node.task.cleanup.time" (in seconds) property. The property can be set at Node startup
+by adding "-d proactive.node.task.cleanup.time=30" (set to 30 seconds as example).
+
+What happens in detail:
+With a 30 seconds termination timeout following steps happen.
+First: When the currently running task is killed (through the REST API or the Scheduler Web Portal a
```

Comment:
Looks good. 
But it also depends on the Run As Me and we don't know what happens there yet.
Furthermore we have that behavior investigated in Bash and Jython. Claiming here that the behavior is always the same will not help us.